info/diktat-kotlin-coding-style-guide-en.md

```diff
@@ -130,6 +130,19 @@ No special prefix or suffix should be used in these names. For example, the foll
 ```kotlin
      @Test fun `my test`() { /*...*/ }
 ``` 
+6. The following table contains some confusing characters. You should be careful when using them as identifiers. 
```

Comment:
Please create an issue that this rule needs to be implemented in identifier naming.